src/qml/imports/QFieldControls/+Qt5/QFieldCamera.qml

```diff
@@ -308,56 +366,58 @@ Popup {
               return Theme.getThemeVectorIcon('ic_flash_off_black_24dp');
             default:
               return'';
+            }
           }
-        }
-        iconColor: "white"
-        bgcolor: Qt.hsla(Theme.darkGray.hslHue, Theme.darkGray.hslSaturation, Theme.darkGray.hslLightness, 0.5)
-        round: true
+          iconColor: "white"
+          bgcolor: Qt.hsla(Theme.darkGray.hslHue, Theme.darkGray.hslSaturation, Theme.darkGray.hslLightness, 0.5)
+          round: true
 
-        onClicked: {
-          if (camera.flash.mode == Camera.FlashOff) {
-            camera.flash.mode = Camera.FlashOn;
-          } else {
-            camera.flash.mode = Camera.FlashOff
+          onClicked: {
+            if (camera.flash.mode == Camera.FlashOff) {
+              camera.flash.mode = Camera.FlashOn;
+            } else {
+              camera.flash.mode = Camera.FlashOff
```

Comment:
```suggestion
           }
           else if (camera.flash.mod == Camera.FlashOn)
             camera.flash.mode = Camera.FlashAuto;
            } else {
              camera.flash.mode = Camera.FlashOff;
```

Otherwise I can't find where the FlashAuto mode is set

Reply:
@domi4484 , the idea here was to only allow for On/Off, hence why Auto isn't set. We can tweak in a separate PR if that's something you feel is needed.